doc/classes/ConfigFile.xml

```diff
@@ -164,6 +164,17 @@
 			<description>
 				Loads the config file specified as a parameter. The file's contents are parsed and loaded in the [ConfigFile] object which the method was called on.
 				Returns [constant OK] on success, or one of the other [enum Error] values if the operation failed.
+				[b]Warning:[/b] Loading Data from a Config File may allow for remote code execution. This is non-trivial to midigate, but storing a [readme] file in the same directory can warn users against downloading untrusted configurations:
+				[codeblocks]
+				[gdscript]
+				var config_path = "user://"
+				var config_file_name = "my_game.config"
+				# check if configuration has been initialized:
+				if not FileAccess.file_exists("%s%s" % [config_path, config_file_name]):
+					var readme := FileAccess.open("%s%s" % [config_path, "README.txt"], FileAccess.WRITE)
+					readme.store_string("Caution: It is not advised to use Config-files you find online, as malicious files may compromise your device. ")
+				[/gdscript]
+				[/codeblocks]
```

Comment:
I am not really happy with this. Feel encouraged to suggest a different solution.

My reasoning for including a code block here btw is making this as easy as possible for potentially lazy developers :P

Reply:
```suggestion
				[b]Warning:[/b] Loading Data from a Config File may allow for remote code execution. This is non-trivial to midigate, but storing a [readme] file in the same directory can warn users against downloading untrusted configurations:
				[codeblocks]
				[gdscript]
				var config_path = "user://"
				var config_file_name = "my_game.config"
				# check if configuration has been initialized:
				if not FileAccess.file_exists("%s%s" % [config_path, config_file_name]):
				    var readme := FileAccess.open("%s%s" % [config_path, "README.txt"], FileAccess.WRITE)
				    readme.store_string("Caution: It is not advised to use Config-files you find online, as malicious files may compromise your device. ")
				    readme.close()
				[/gdscript]
				[/codeblocks]
```

Reply:
Fixed 2 out of 3 typos. I do not understand the issue that was raised by "readme".

Reply:
You can't use the brackets for classes that don't exist, you should use `[code]`:
```suggestion
				[b]Warning:[/b] Loading Data from a Config File may allow for remote code execution. This is non-trivial to midigate, but storing a [code]readme[/code] file in the same directory can warn users against downloading untrusted configurations:
				[codeblocks]
				[gdscript]
				var config_path = "user://"
				var config_file_name = "my_game.config"
				# Check if configuration has been initialized:
				if not FileAccess.file_exists("%s%s" % [config_path, config_file_name]):
					var readme = FileAccess.open("%s%s" % [config_path, "README.txt"], FileAccess.WRITE)
					readme.store_string("Caution: It is not advised to use Config-files you find online, as malicious files may compromise your device.")
				[/gdscript]
				[/codeblocks]
```
Also we [don't add](https://docs.godotengine.org/en/stable/contributing/documentation/docs_writing_guidelines.html#dynamic-vs-static-typing) static typing to the documentation